src/shipit_frontend/src/App.elm

```diff
@@ -28,28 +29,29 @@ type Page
     | ReleaseDashboard
 
 
-type alias Model =
-    { release_dashboard : ReleaseDashboard.Model
-    , current_page : Page
-    , current_user : Maybe User.Model
-    , dashboard_url: String
-    }
+type alias Model = {
+  release_dashboard : ReleaseDashboard.Model,
+  current_page : Page,
+  current_user : Maybe User.Model,
+  backend_url: String
+}
 
 type Msg
     = ShowPage Page
     | UserMsg User.Msg
     | ReleaseDashboardMsg ReleaseDashboard.Msg
+    | SelectAnalysis ReleaseDashboard.Analysis
 
 type alias Flags = {
-    user : Maybe User.Model,
-    dashboardUrl : String
+    backend_url : String
```

Comment:
I changed this variable to backend_dashboard_url in c6273dddaac780f0597d01000cbfc15bd0bc9b38